lua/nvim-lsp-installer/servers/ocamlls/init.lua

```diff
@@ -5,6 +5,10 @@ return function(name, root_dir)
     return server.Server:new {
         name = name,
         root_dir = root_dir,
+        deprecated = {
+            message = "ocamlls is deprecated, use ocamllsp instead.",
+            replace_with = "ocamllsp",
+        },
```

Comment:
This will show up in the UI:

<img width="568" alt="Screenshot 2022-02-21 at 22 05 18" src="https://user-images.githubusercontent.com/6705160/155026157-6700e1c4-90a7-49bc-b3dc-c46ec29b2697.png">

Reply:
Nice!